Sources/BraintreeVisaCheckout/BTVisaCheckoutNonce.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+#if canImport(BraintreeCard)
+import BraintreeCard
+#endif
+
+@objc public class BTVisaCheckoutNonce: BTPaymentMethodNonce {
+
+    @objc public let lastTwo: String
+    @objc public let cardType: String
+    @objc public let billingAddress: BTVisaCheckoutAddress
+    @objc public let shippingAddress: BTVisaCheckoutAddress
+    @objc public let userData: BTVisaCheckoutUserData
+    @objc public let callId: String
+    @objc public let binData: BTBinData
+
+    @objc public init(
```

Comment:
This can be internal. We can also just init with `BTJSON` and extract each of these from the JSON. You can take a look at other nonce classes as a reference.

Reply:
We will also need to add the parsing logic to the init. Can use Android as a reference and unit tests to verify.